ext/span.h

```diff
@@ -92,6 +92,8 @@ struct ddtrace_span_data {
         ddtrace_span_properties;
         ddtrace_span_properties props;
     };
+
+    zval property_baggage;
```

Comment:
It belongs at the end of ddtrace_span_properties, not after ddtrace_span_properties here.